spec/menu_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'menu'
+
+describe Menu do
+
+  let(:menu) { Menu.new }
+
+  describe '#load_menu' do
+
+    it 'should load menu from csv file into @menu' do
```

Comment:
Does this specific test test for the existence of a CSV file, or is a further test needed to clarify that the menu is being loaded from a CSV file?